python/peacock/ExodusViewer/plugins/MeshPlugin.py

```diff
@@ -239,15 +376,17 @@ def main(size=None):
     """
     from peacock.ExodusViewer.ExodusPluginManager import ExodusPluginManager
     from peacock.ExodusViewer.plugins.VTKWindowPlugin import VTKWindowPlugin
-    widget = ExodusPluginManager(plugins=[lambda: VTKWindowPlugin(size=size), MeshPlugin])
+    from peacock.ExodusViewer.plugins.FilePlugin import FilePlugin
+    widget = ExodusPluginManager(plugins=[lambda: VTKWindowPlugin(size=size), FilePlugin, MeshPlugin])
     widget.show()
 
     return widget, widget.VTKWindowPlugin
 
 if __name__ == '__main__':
     from peacock.utils import Testing
     app = QtWidgets.QApplication(sys.argv)
-    widget, window = main()
-    filename = Testing.get_chigger_input('mug_blocks_out.e')
-    window.initialize([filename])
+    filenames = Testing.get_chigger_input_list('mug_blocks_out.e', 'displace.e')
+    #filenames = Testing.get_chigger_input_list('diffusion_1.e', 'diffusion_2.e')
```

Comment:
remove?

Reply:
I would like to keep this there, this is for debugging.